WordPress/Classes/Networking/ServiceRemoteXMLRPC.m

```diff
@@ -22,16 +22,12 @@ - (id)initWithApi:(WPXMLRPCClient *)api username:(NSString *)username password:(
     return self;
 }
 
-- (NSArray *)getXMLRPCArgsForBlogWithID:(NSNumber *)blogID extra:(id)extra
-{
-    NSMutableArray *result = [NSMutableArray array];
-    NSString *password = self.password ?: [NSString string];
-    NSString *username = self.username ?: [NSString string];
-    
-    [result addObject:blogID];
-    [result addObject:username];
-    [result addObject:password];
-    
+- (NSArray *)defaultXMLRPCArguments {
+    return @[@0, self.username, self.password];
```

Comment:
I know what the 0 is all about, but maybe add a comment for posterity? The reasons might not be obvious to newer devs.